build/build.go

```diff
@@ -575,7 +575,7 @@ func NewMappingCallback(m *sourcemap.Map, goroot, gopath string) func(generatedL
 	}
 }
 
-func jsFilesFromDir(dir string) ([]string, error) {
+func JsFilesFromDir(dir string) ([]string, error) {
```

Comment:
I think we should apply the standards and guidelines of the Go community. If existing code can be improved, we should do so. `JsFiles` should clearly become `JSFiles`. Some more documentation would also be nice, however in this case I would like to avoid giving the impression that the `build` package is stable API. It does not feel ready to me. If we add more docs then we should warn in the package description about potential future changes.

Speaking of changes, making `jsFilesFromDir` exported seems wrong to me. This is an internal helper, the package should make all necessary cases available via higher-level API. Also adding this code do `collectTests` will probably include the `.inc.js` files twice.

The root cause is bad API of the `build` package. `jsFilesFromDir` should probably somehow move into the `Import` function and then there should also be an `ImportDir` function. It also occurred to me that `Session.ImportPackage` is probably a bad name, since it actually builds the package. Seems like I did not do a good job at this part of the codebase and it needs some refactoring to become nice.